intelmq/bots/BOTS

```diff
@@ -500,6 +500,13 @@
             "module": "intelmq.bots.experts.gethostbyname.expert",
             "parameters": {}
         },
+        "IDEA": {
+            "description": "Converts events into the IDEA format.",
+            "module": "intelmq.bots.experts.idea.expert",
+            "parameters": {
+                "test_mode": false
```

Comment:
The additional Test category is the standard practice in Idea/Warden community (as are multiple categories). Is there any other more standard or general way to put in two categories in IntelMQ?